tripal/src/Plugin/Field/FieldType/TripalBooleanTypeItem.php

```diff
@@ -24,6 +24,36 @@ class TripalBooleanTypeItem extends TripalFieldItemBase {
 
   public static $id = "tripal_boolean_type";
 
+  /**
+   * {@inheritdoc}
+   */
+  public static function defaultFieldSettings() {
+    $settings = [];
+    return $settings + parent::defaultFieldSettings();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function defaultStorageSettings() {
+    $settings = [
+      'storage_plugin_id' => 'drupal_sql_storage',
+    ];
+    return $settings + parent::defaultStorageSettings();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function generateSampleValue(FieldDefinitionInterface $field_definition) {
+    $values = [];
+
+    $random = new \Drupal\Component\Utility\Random();
+    $values['value'] = TRUE;
```

Comment:
Do we want to generate TRUE half the time and FALSE half the time here?

Reply:
I thought about this but must have forgotten before taking the time to actually implement it 🙈 This is now fixed in d064e4374.